romanesco/format/__init__.py

```diff
@@ -222,14 +206,10 @@ def get_analysis(filename):
 
         for filename in validator_files:
             analysis = get_analysis(filename)
-            in_type = analysis["inputs"][0]["type"]
-            in_format = analysis["inputs"][0]["format"]
 
-            conv_graph.add_node(Validator(in_type, in_format), {
-                "type": in_type,
-                "format": in_format,
-                "validator": analysis
-            })
+            conv_graph.add_node(Validator(analysis["inputs"][0]["type"],
+                                          analysis["inputs"][0]["format"]),
+                                analysis)
```

Comment:
Validators are a special case only ever taking one input, and only ever returning one output. I'm not sure how we would handle that type of API for multiple inputs/outputs analyses in general.

Regardless I'm adding a comment noting the magic numbers.